src/screens/LocationScreen.js

```diff
@@ -86,48 +98,83 @@ class LocationScreen extends Component {
 
   onPress = async () => {
     const { status, ios } = await Location.requestPermissionsAsync();
-    if (status !== 'granted') {
+    if (status !== "granted") {
       this.setState({
-        errorMessage: 'Permission to access location was denied',
+        errorMessage: "Permission to access location was denied"
       });
     }
-    if (status === 'granted') {
-      await Location.startLocationUpdatesAsync(
-        TASK_GUARDIAN_LOCATION, {
-          accuracy: Location.HIGH,
-          showsBackgroundLocationIndicator: true,
-          //timeInterval: 0,
-          distanceInterval: '20', // meters
-          deferredUpdatesInterval: '200', //ms
-          deferredUpdatesDistance: '20', //meters
-          pausesUpdatesAutomatically: true,
-      })
+    if (status === "granted") {
+      await Location.startLocationUpdatesAsync(TASK_GUARDIAN_LOCATION, {
+        accuracy: Location.HIGH,
+        showsBackgroundLocationIndicator: true,
+        //timeInterval: 0,
+        distanceInterval: "20", // meters
+        deferredUpdatesInterval: "200", //ms
+        deferredUpdatesDistance: "20", //meters
+        pausesUpdatesAutomatically: true
+      });
     }
   };
 
   render() {
-    let text = 'Waiting..';
+    let text = "Waiting..";
     if (this.state.errorMessage) {
       text = this.state.errorMessage;
     } else if (this.state.location) {
       text = JSON.stringify(this.state.location);
     }
-
-    return (
-      <View style={styles.container}>
+    let phoneOrLocRequest = "Loading...";
+    // ***** Set to "not" for testing, should be changed back for production *****
+    if (!this.state.location) {
+      phoneOrLocRequest = (
         <TouchableOpacity onPress={this.onPress}>
           <Text>Enable background location</Text>
         </TouchableOpacity>
-        <Text style={styles.paragraph}>{text}</Text>
-      </View>
+      );
+    } else {
+      phoneOrLocRequest = (
+        <View>
+          <Text style={styles.instructions}>Phone Number</Text>
+          <TextInput
+            style={{
+              width: 250,
+              height: 40,
+              borderColor: "gray",
+              borderWidth: 1,
+              marginBottom: 10,
+              alignSelf: "center",
+              margin: "auto"
+            }}
+            autoCompleteType="tel"
+            type="tel"
+            placeholder="+1(555)555-555"
+            onChangeText={text => (this.state.inputText = text)}
```

Comment:
This is fine, but it would be better to dispatch into the store. 

See src/screens/ProfileScreen.js for example dispatch:

dispatch(setUserPhone(phoneNumber))